retriever/engines/postgres.py

```diff
@@ -17,21 +18,27 @@ class engine(Engine):
         "bool": "boolean",
     }
     max_int = 2147483647
+    pgpass_file = io.open(PGPASS_FILE_PATH, 'r')
```

Comment:
I prefer using the custom open function for the retriever in ` __init__.py`
before we open the file, we should check if path exists, and if it does exist we could update else we should stick to the default(no updates to the dictionary)